### PR TITLE
Adjust booleans in system modules

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
https://github.com/ansible-community/community-topics/issues/116

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
various

### CODE REVIEW

File: plugins/modules/system/ufw.py [hunk 1]
```diff
@@ -285,7 +285,7 @@
 - name: Deny forwarded/routed traffic from subnet 1.2.3.0/24 to subnet 4.5.6.0/24
   community.general.ufw:
     rule: deny
-    route: yes
+    route: true
     src: 1.2.3.0/24
```
Comment (samccann) [inline plugins/modules/system/ufw.py:289]:
```suggestion
    src: 192.0.2.0/24
```

File: plugins/modules/system/ufw.py [hunk 2]
```diff
@@ -285,7 +285,7 @@
...
     dest: 4.5.6.0/24
```
Comment (samccann) [inline plugins/modules/system/ufw.py:290]:
```suggestion
    dest: 198.51.100.0/24
```

File: plugins/modules/system/ufw.py (hunk 1 above)
Comment (samccann) [reply plugins/modules/system/ufw.py:289]: strictly speaking, we shouldn't use real IP addresses. 1.2.3.0 is actually owned by someone ;-) The two replacements here are documentation reserved IP addresses per https://www.rfc-editor.org/rfc/rfc5737

**Review by samccann — APPROVED** [review]
Added stray IP address nits. Feel free to ignore if you want to keep this PR clean w/ just boolean fun.

File: plugins/modules/system/ufw.py (hunk 1 above)
Comment (felixfontein) [reply plugins/modules/system/ufw.py]: I'm sure there are more IP addresses all over the collection's examples... If someone is sufficiently bored, they can go searching and replacing ;-) (and I don't want to imply that this hypothetical someone is you :) ) I've added these two changes to the PR.

**Comment by felixfontein** [comment]
@samccann thanks for reviewing this!

**Pull request merged**